WooCommerce/Classes/POS/Presentation/PaymentButtons.swift

```diff
@@ -94,7 +94,8 @@ private extension PaymentsActionButtons {
         couponsController: PointOfSalePreviewCouponsController(),
         cardPresentPaymentService: CardPresentPaymentPreviewService(),
         orderController: PointOfSalePreviewOrderController(),
-        collectOrderPaymentAnalyticsTracker: POSCollectOrderPaymentAnalytics())
+        collectOrderPaymentAnalyticsTracker: POSCollectOrderPaymentAnalytics(),
```

Comment:
We deserve a single definition of PointOfSaleAggregateModel preview 😀

Reply:
Yeah, I thought that as I pushed this branch! I'll do it next time.